ollama/_client.py

```diff
@@ -90,11 +90,13 @@ def __init__(
       base_url=_parse_host(host or os.getenv('OLLAMA_HOST')),
       follow_redirects=follow_redirects,
       timeout=timeout,
+      # Lowercase all headers to ensure override
       headers={
-        'Content-Type': 'application/json',
-        'Accept': 'application/json',
-        'User-Agent': f'ollama-python/{__version__} ({platform.machine()} {platform.system().lower()}) Python/{platform.python_version()}',
-      }.update(headers or {}),
+        **{k.lower(): v for k, v in (headers or {}).items()},
```

Comment:
Had to run `.lower()` since python would keep both keys but during a http request they would double - so jank

Reply:
instead of lowering `headers` keys, you can simplify this slightly by lowering the final dict before passing to httpx. that will be less error prone since the client-set headers will also be case insensitive 

```python
self._client = client(
    headers={
        k.lower(): v for k, v in {
            **(headers or {}),
            'Content-Type': 'application/json',
            'Accept': 'application/json',
            'User-Agent': f'...',
        }
    },
)
```

Reply:
The merging of the dicts inside the `.lower` would be case based which means that we'd still get unwanted behavior. Not sure how what the new dict would look like since there would be key collision after the `.lower()`